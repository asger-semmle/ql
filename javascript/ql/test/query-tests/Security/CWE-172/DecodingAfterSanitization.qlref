Security/CWE-172/DecodingAfterSanitization.ql
