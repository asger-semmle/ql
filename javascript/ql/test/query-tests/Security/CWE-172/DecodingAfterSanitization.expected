| tst.js:12:26:12:32 | nextUrl | Decoding invalidates prior sanitization performed $@. | tst.js:8:8:8:26 | isValidUrl(nextUrl) | here |
| tst.js:17:27:17:30 | data | Decoding invalidates prior sanitization performed $@. | tst.js:16:14:16:37 | _.escap ... y.data) | here |
